feature-libs/product-configurator/rulebased/styles/_configurator-show-more.scss

```diff
@@ -13,6 +13,10 @@
         color: var(--cx-color-primary);
         text-decoration: underline;
       }
+
+      &:focus {
```

Comment:
You don't need to add a focus to the button. The button has a focus out of the box. It was disabled: `outline: none;`
You had to remove the line and it would work as expected.

Reply:
I did it and added a text-decoration